src/main/java/com/aetherteam/aether_genesis/block/GenesisBlocks.java

```diff
@@ -55,6 +56,9 @@ public class GenesisBlocks {
     public static final RegistryObject<WallBlock> SKYROOT_WOOD_WALL = register("skyroot_wood_wall", () -> new GenesisDoubleDropsWall(Block.Properties.of(Material.WOOD, MaterialColor.WOOD).strength(2.0F).sound(SoundType.WOOD)));
     public static final RegistryObject<WallBlock> STRIPPED_SKYROOT_WOOD_WALL = register("stripped_skyroot_wood_wall", () -> new WallBlock(Block.Properties.of(Material.WOOD, MaterialColor.WOOD).strength(2.0F).sound(SoundType.WOOD)));
 
+    public static final RegistryObject<Block> AETHER_CRAFTING_TABLE = register("skyroot_crafting_table", () -> new BlockAetherCraftingTable(Block.Properties.of(Material.WOOD, MaterialColor.WOOD).strength(2.5F).sound(SoundType.WOOD)));
+    public static final RegistryObject<Block> HOLYSTONE_FURNACE = register("holystone_furnace", () -> new BlockHolystoneFurnace(Block.Properties.of(Material.STONE, MaterialColor.STONE).strength(2.5F)));
```

Comment:
needs .requiresCorrectToolForDrops()

Reply:
> * Warped Chocolates need to be renamed to Wrapped Chocolates.
> * Zephyr Mixin is probably unnecessary, but it will require some design discussion.
> * I still think you should try to make Tempest code extend Zephyr code as much as possible (where doable at least) for models, entity classes, etc.

i'll fix them all, thank you for time to check this